.github/workflows/update-getting-started-guide.yml

```diff
@@ -0,0 +1,56 @@
+name: Update getting started guide
+
+on:
+  release:
+    types: ['released']
+  pull_request:
+    branches: ['main', 'release/**']
+    paths:
+      - '.github/workflows/update-getting-started-guide.yml'
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-version:
+    runs-on: ubuntu-20.04
+    env:
+      RELEASE_TAG: ''
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Mock release tag on pull request
+        if: github.event_name == 'pull_request'
```

Comment:
Why do we need to run this on pull requests?

Reply:
Enables spot checking of any changes to the workflow in pull request. It won't catch every issue, but it can give some confidence.

Reply:
My understanding is that, on PR, it will create a dummy tag, replace the tag with the dummy tag, and diff check it, but ultimately not fail for any particular reason. What errors are we expecting this to catch?

Reply:
e.g. changes to the getting started guide layout which would break the update workflow.

Reply:
Added run on pull request to docs/getting-started.md in latest commit. PTAL.